tests/test_quixstreams/test_dataframe/test_dataframe.py

```diff
@@ -1676,8 +1683,11 @@ def test_group_by_column(
         sdf[col] = col_update
 
         groupby_topic = sdf.topics[0]
-        assert sdf_registry.consumer_topics == [topic, groupby_topic]
-        assert groupby_topic.name.startswith("repartition__")
+        if num_partitions == 1:
+            assert sdf_registry.consumer_topics == [topic]
```

Comment:
You can move `groupby_topic = sdf.topics[0]` to else clause or add `assert groupby_topic == topic` to if clause.
```suggestion
            assert sdf_registry.consumer_topics == [topic]
            assert groupby_topic == topic
```